### ISSUE TITLE
compose reply message always cc's myself

### ISSUE THREAD

**Issue by uliw** [djcb/mu#1668]
## Expected or desired behavior
there are numerous posted solutions around this issue. Based on these suggestions I have the following setup:
(setq user-mail-address "uli.wortmann@utoronto.ca")
before loading mu4e
(setq user-full-name  "Ulrich G. Wortmann")
(setq mu4e-compose-dont-reply-to-self t)
(setq mu4e-user-mail-address-list (quote ("uli.wortmann@utoronto.ca")))
Based on these settings, I would expect that hitting R with "reply to all" would generate a message window where my own e-mail has been stripped from the CC list

## Actual behavior
My own e-mail is always included in the CC-list, as 
"Ulrich G. Wortmann" <uli.wortmann@utoronto.ca>.
Since I am cc'ed on the reply, the reply will show up as a new message in the header buffers.
In other words, the message thread re-appears, even so, no response has been received from the other party yet.  The desired behavior is that only actual responses are shown.

## Steps to reproduce
this happens every time on my machine, but maybe set up specific

## Versions of mu, mu4e/emacs, operating system etc.
mu4e 1.4.1, using gmail via imap, emacs version is 26.3

## Any other detail
I'd be grateful for any pointers on how to look into this

Cheers

Uli

**Comment by jschwab** [comment]
1.4.x no longer uses `mu4e-user-mail-address-list`.

See the NEWS: https://github.com/djcb/mu/blob/master/NEWS.org#mu4e

I think that is a likely the source of the issue.

**Comment by uliw** [comment]
Hooray!

that did it! 

Thanks so much!

Uli

Josiah Schwab <notifications@github.com> writes:

> 1.4.x no longer uses `mu4e-user-mail-address-list`.
>
> See the NEWS: https://github.com/djcb/mu/blob/master/NEWS.org#mu4e
>
> I think that is a likely the source of the issue.


-- 
Ulrich G. Wortmann          http://www.es.utoronto.ca/people/faculty/wortmann-ulrich/
Dept. of Earth Sciences     Fax  : 416 978 3938
University of Toronto       Phone: 416 978 7084
22 Russell Street, Toronto, ON, Canada M5S 3B1
